packages/fhir-converter/src/lib/workers/worker.js

```diff
@@ -13,7 +13,8 @@ var errorMessage = require("../error/error").errorMessage;
 var HandlebarsConverter = require("../handlebars-converter/handlebars-converter");
 var WorkerUtils = require("./workerUtils");
 var dataHandlerFactory = require("../dataHandler/dataHandlerFactory");
-var { extractEncounterTimePeriod } = require('../inputProcessor/dateProcessor');
+var { extractEncounterTimePeriod } = require("../inputProcessor/dateProcessor");
+var { uuidv7 } = require("@metriport/core/util/uuid-v7");
```

Comment:
is this the first time we are introducing a dependency in the fhir-converter to other packages? 
If so, maybe we can just import this from the uuid library itself so we dont have that dependency?

Reply:
good point. Replaced with uuidv4 and tested that it still generates the ID and passes it properly thru the conversion